### ISSUE TITLE
cli missing code samples

### ISSUE THREAD

**Issue by jamesatha** [algorithmiaio/api-docs#18]
When you visit http://docs.algorithmia.com/?cli there are a few code sections that are blank:
`Updating a directory:`
`Checking if a file exists:`

Not sure if we can remove those sections, update the CLI so it can handle these cases, or just document that it's not supported right now

**Comment by anowell** [comment]
yes. those are currently unsupported... I imagine something like:

- `algo chmod <dir>` would be the `update` case.
- `algo stat <file>` would be the way to figure out if a file exists.

But until such features are implemented, documenting them as unsupported sounds reasonable
